README.md

```diff
@@ -40,6 +40,19 @@ Read the [Roadmap](./ROADMAP.md).
 
 Read the [Deploy TiDB using Kubernetes on Your Laptop for development and testing](./docs/local-dind-tutorial.md).
 
+[![Open in Cloud Shell](https://gstatic.com/cloudssh/images/open-btn.png)](https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/pingcap/tidb-operator)
```

Comment:
If it is on ./docs/google-kubernetes-tutorial.md, then it also shows within the tutorial (which is confusing for novices because you already have it open!)

An alternative would be to create ./docs/google-kubernetes-overview.md which summarizes the tutorial briefly and includes the button.

Do you have a preference?

Reply:
I see. A bigger problem right now is actually that users may not understand what the button really does.

How about: Click this <button> to launch our [Google Cloud Platform Kubernetes tutorial](./docs/google-kubernetes-tutorial.md) as a Google Cloud Shell tutorial.

If the button doesn't work, someone can come back to click the link here, and the linked tutorial can give the manual `teachme` technique.

Reply:
I researched to see if there was a way to conditionally show the launch button when not in tutorial mode (i.e. someone stumbled across the `.md` document).  This would make the most sense, but it doesn't appear to be possible :(  I looked at some other Cloud Shell tutorials, and they don't have the launch button inside them.

I've reduced the screen real estate on `README.md`.  There is not a suggestion on how to open outside of Google Shell though: I think we should cater to the common case for readability.  Advanced users can substitute if they desire.